utils/goDotEnv.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GoDotEnvVariable(key string) string {
+
+	err := godotenv.Load(".env")
+
+	if err != nil {
+		log.Println("Error loading .env file")
+	}
```

Comment:
If there is no .env file, then there will be no credentials set for other code to function correctly, Is it reasonable to proceed further here?

Reply:
No @prakashchoudhary07 we need to throw error to the users.
I'm thinking to use log.Panic() so that the program doesn't get end abruptly.
What are your thoughts?